### DISCUSSION TITLE
Save a track for later use / Recent tracks list

### DISCUSSION THREAD

**Discussion by kcpr** [organicmaps/organicmaps#2741] [Ideas]
Hi,

I'm really missing an option to save track for later use and quickly reopen recently prepared tracks. I think it's especially important when I leave track creation mode for some reason - even accidentally - and already had few stops set (without it it's just irritating sometimes to be honest).

It may be connected with #2384 as KML could be used to achieve this I believe. If that would be implemented, just a bookmark button in track creation mode to save a track or share button to export it to a file could be enough.
On the other hand I believe just some internal way of saving it may be easier to implement and would be enough - with "recent tracks" list. 

Thank You for the application by the way and for that feature in advance!

**Comment by biodranik** [comment]
I think it was already mentioned in some issues. Can you please double-check and create an issue if there is no one?

**Reply by kcpr** [reply]
Thank You. I just checked the issues list and discussions and the closes one I've found is this: https://github.com/organicmaps/organicmaps/discussions/781 . It is similar indeed, but:
1. As I noted just saving track to a bookmark will still require #2384 I assume.
2. No "recent tracks" list idea has been mentioned there.

I can anyway present my ideas there if that's more comfortable (or easier to manage) to You.

**Reply by kcpr** [reply]
Or we could just assume this conversation is strictly about "recent tracks list".

**Reply by kcpr** [reply]
Oh, sorry, I've just discovered You meant Issues strictly and wanted me to create one. I'll then.

**Reply by kcpr** [reply]
I'm not sure though: should I create two issues out of that or just one? I could use this one for "recent tracks list" one and https://github.com/organicmaps/organicmaps/discussions/781 for "saving route to a bookmark" if that's ok for You.
(And sorry for that many messages.)
